app/assets/javascripts/miq_application.js

```diff
@@ -1540,7 +1548,7 @@ function miqToolbarOnClick(_e) {
       tb_url += "/" + ManageIQ.record.recordId;
     }
     tb_url += "?pressed=";
-    if (typeof button.data('pressed') == "undefined") {
+    if (typeof button.data('pressed') == "undefined" && button.data('click')) {
```

Comment:
This is almost certainly wrong, because if pressed *was* undefined, then you'll get `?pressed=undefined` in `tb_url`... (`"" + undefined === "undefined"`)